### ISSUE TITLE
Parinaud oculoglandular syndrome

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10614]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Mannis MJ, Holland EJ. Cornea. 4th ed. St. Louis, Mo: Elsevier Inc.; 2015:518-525. : Parinaud oculoglandular syndrome is a conjunctival disease characterized by a unilateral granulomatous follicular conjunctivitis associated with ipsilateral regional lymphadenopathy. There are many vectors that transmit organisms that cause oculoglandular syndrome, of which, Cat Scratch Disease is the most common cause. Prognosis in most cases is excellent.

PMID: 29489252 Ophthalmic complications include neuroretinitis, papillitis, optic neuritis, and most commonly Parinaud Oculoglandular syndrome. This is an atypical presentation found in 2% to 8% percent of children, which involves an inoculation site in the conjunctiva. There is local adenopathy, typically preauricular, upper cervical, or submandibular, with severe conjunctival infection. The conjunctival infection resolves in several weeks, but the lymphadenopathy may persist for months. 

PMID: 21243990 Parinaud oculoglandular syndrome is the most common ocular manifestation and consists of granulomatous conjunctivitis and ipsilateral periauricular lymphadenopathy

**Parent term (use hpo.jax.org/app)**
Lymphadenopathy HP:0002716
Conjunctivitis HP:0000509

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cat-scratch disease ORPHA:50839
Tularemia  ORPHA:3392
ORPHA:826 Sporotrichosis

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
It seems to me that "Parinaud oculoglandular syndrome" is a disease rather than a phenotypic feature, and as such it should get an ORPHA code rather than an HPO code!

**Comment by carolina-f** [comment]
As the definition says, Parinaud oculoglandular syndrome can be caused by various organisms, of which Bartonella is the most common, but not the only one.
This syndrome always has a causal agent and I do not consider it to be a disease in itself, but rather as a sign accompaniment in the clinical picture of another disease, as is the case of Horner syndrome.
PMID: 32751142
(….) We found multiple cases due to cat scratch disease (CSD; due to Bartonella henselae) (12), tularemia (10), sporotrichosis (3), Rickettsia conorii (3), R. typhi/felis (2), and herpes simplex virus (2) and single cases due to tuberculosis, paracoccidioidomycosis, Yersinia enterocolitica, Pasteurella multocida, Chlamydia trachomatis, Epstein–Barr virus, and Nocardia brasiliensis. Preauricular lymphadenopathy is a common clinical clue for Parianud OculoGlandular Syndrome and is unusual in viral and bacterial conjunctivitis

**Comment by carolina-f** [comment]
what do you think about it?

**Comment by pnrobinson** [comment]
reopening to discuss.

**Comment by pnrobinson** [comment]
According to the definition, "Parinaud's oculoglandular syndrome (POGS) is defined as unilateral granulomatous conjunctivitis and facial lymphadenopathy. " -- it can occur as a complication of other diseases. This is one of the items that is on the boundary between being a disease entity with a very specific cause and a phenotypic feature of disease. Given that it is a feature mainly of various infectious diseases --- bacterial, fungal, and viral infections, but most commonly in cat scratch disease (CSD) , if we make this an HPO term it will be difficult to find for users because I believe neither the ORPHA annotations nor our annotations feature these diseases. Since it is a rare complication of these diseases, probably it would be better to make it an ORPHA term so that patients and doctors could find it. 
In any case, I think in the HPO system it would be possible to annotate the individual features of Parinaud, and that would be the prefered way of doing things (ORPHA entry for Parinaud's oculoglandular syndrome as a disease where is it noted to be linked to the other diseases, and HPO terms to represent the phenotypic features).

Does this sound reasonable?

**Comment by carolina-f** [comment]
For the moment, I have made the request for the new terms "granulomatous follicular conjunctivitis" and "preauricular lymphadenopathy".
